contracts/deployless/Estimation.sol

```diff
@@ -138,12 +139,19 @@ contract Estimation is FeeTokens, Spoof {
     address relayer,
     address oracle
   ) external returns (EoaEstimationOutcome memory eoa) {
-    // simulate the transactions
     SimulationOutcome memory simulation;
-    (simulation, , ) = simulateUnsigned(op, associatedKeys);
```

Comment:
this seems like an un-elegant way to do things

why don't we introduce per-call limits in the `simulation` itself and not touch the top-level code

or just, considering you don't use those limits, drop/revert this altogether